docs/source/algorithms/MaskDetectors-v1.rst

```diff
@@ -132,13 +130,14 @@ mask detectors, including
 
 -  Workspace indices
 -  Spectra
--  Detectors
+-  Detects
```

Comment:
Thanks for checking the documentation as well! Indeed, 'Detector IDs' would be a better fit here, also perhaps 'Spectrum numbers' instead of 'Spectra'. I did play with the idea here, thus the unindented typo 'Detects'.